File: main workshop/services/dotnet/payment-processor/Models/PaymentGatewayResult.cs
using System.Text.Json.Serialization;

public class PaymentGatewayResult
{
    [JsonPropertyName("success")]
    public bool Success { get; set; }
    [JsonPropertyName("transactionId")]
    public string? TransactionId { get; set; }
    [JsonPropertyName("errorMessage")]
    public string? ErrorMessage { get; set; }
    [JsonPropertyName("processingTimeMs")]
    public long ProcessingTimeMs { get; set; }
}

File: main-workshop/services/java/payment-processor/src/main/java/com/powertoolsride/paymentprocessor/model/PaymentGatewayResult.java
package com.powertoolsride.paymentprocessor.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public record PaymentGatewayResult(
    @JsonProperty("success") boolean success,
    @JsonProperty("transactionId") String transactionId,
    @JsonProperty("errorMessage") String errorMessage,
    @JsonProperty("processingTimeMs") long processingTimeMs
) {}
